components/eam/bld/build-namelist

```diff
@@ -1229,59 +1307,62 @@ if ($aer_model eq 'mam' ) {
     @mode_types    = qw(accum aitken coarse primary_carbon);
     @mode_num      = qw(num_a1 num_a2 num_a3 num_a4);
     @mode_num_cw   = qw(num_c1 num_c2 num_c3 num_c4);
+
+#++hybrown
+# Change with changes to number of tagged species
     if ($rain_evap_to_coarse_aero == 0) { #default
 	@mode_spec     = (
-	    [qw(so4_a1 pom_a1 soa_a1 bc_a1 dst_a1 ncl_a1)],
```

Comment:
All of these changes seem really specific to the exact number of tagged tracers added here. Can these be just namelist edits in a runscript, or can this be generalized somehow? Hard-coding these all in breaks existing cases.